docs/attributes.md

```diff
@@ -0,0 +1,47 @@
+# Object Attributes (attrs)
+
+# This is still a work in progress (mistakes are common).
+
+## For objects that have associated events
+> Required in **bold**
+- duration: number; The number of users that can be admitted in the event associated to the event.
+- eventStartDelay: The number of minutes before the the event starts after the startAt field of the event.
+- eventDuration: number; The duration in minutes of the event associated to the object.
+- groupSize: number; The number of students that are required to be part of a group.
```

Comment:
groupSize is related to raids (to automatic generation of groups to be more precise)
it is not a number required, it is the number wanted (it could be less or more according to the number of students/candidates registered)
for example if the groupSize is 3 and there are 8 users in the event, you'll have 3-3-2